share/spice/detect_lang/detect_lang.js

```diff
@@ -1,149 +1,153 @@
+                it: "Italian",
+                iu: "Inuktitut",
+                ja: "Japanese",
+                ka: "Georgian",
+                km: "Khmer",
+                kn: "Kannada",
+                ko: "Korean",
+                lo: "Laothian",
+                lt: "Lithuanian",
+                lv: "Latvian",
+                mk: "Macedonian",
+                ml: "Malayalam",
+                ms: "Malay",
+                mt: "Maltese",
+                my: "Burmese",
+                nb: "Norwegian",
+                nl: "Dutch",
+                or: "Oriya",
+                pa: "Punjabi",
+                pl: "Polish",
+                pt: "Portuguese",
+                ro: "Romanian",
+                ru: "Russian",
+                si: "Sinhalese",
+                sk: "Slovak",
+                sl: "Slovenian",
+                sq: "Albanian",
+                sr: "Serbian",
+                sv: "Swedish",
+                sw: "Swahili",
+                syr: "Syriac",
+                ta: "Tamil",
+                te: "Telugu",
+                th: "Thai",
+                tr: "Turkish",
+                uk: "Ukrainian",
+                ur: "Urdu",
+                vi: "Vietnamese",
+                yi: "Yiddish",
+                zh: "Chinese"
+            };
 
-        return langs[language] || "";
-    };
-    Handlebars.registerHelper("DetectLang_expandLang", expandLang);
+            return langs[language] || "";
+        };
+        Handlebars.registerHelper("DetectLang_expandLang", expandLang);
 
-    if(expandLang(api_result.data.detections[0].language) === "") {
-        return;
-    }
+        if(expandLang(api_result.data.detections[0].language) === "") {
+            return;
+        }
 
-    var d0 = api_result.data.detections[0],
-        d1 = null;
+        var d0 = api_result.data.detections[0],
+            d1 = null;
 
-    if (api_result.data.detections.length > 1) {
-        d1 = api_result.data.detections[1];
-    }
+        if (api_result.data.detections.length > 1) {
+            d1 = api_result.data.detections[1];
+        }
 
-    // Display the plug-in.
-    Spice.add({
-        id: 'detect_lang',
-        data: { first: d0, second: d1 },
-        name: "Answer",
-        meta: {
-            sourceUrl       : "http://detectlanguage.com/",
-            sourceName      : "Detect Language",
-        },
-	signal: 'high',
-        templates: {
-            group: 'base',
-            options: {
-                content: Spice.detect_lang.content,
-		moreAt: true
+        // Display the plug-in.
+        Spice.add({
+            id: 'detect_lang',
+            data: {
+                first: d0,
+                second: d1
+            },
+            name: "Answer",
+            meta: {
+                sourceUrl: "http://detectlanguage.com/",
+                sourceName: "Detect Language",
+            },
+            signal: 'high',
+            templates: {
+                group: 'base',
```

Comment:
Let's use the `text` template + a `title` and `subtitle`.

We can convert the Handlebars helpers into private functions here and use them if needed. This should simplify things and improve the design to match our other Instant Answers